docs/docusaurus/docs/terms/data_context.md

```diff
@@ -42,7 +42,7 @@ A Data Context includes basic create, read, update, and delete (CRUD) operations
 
 The Data Context doesn't just give you convenient ways to access and configure components.  It also provides the ability to *create* top-level components such as Datasources, Checkpoints, and Expectation Suites and manage where the information about those components is stored.  
 
-For production deployments you will want to define these components according to your source data systems and production environment.  This may include storing information about those components in something other than your local environment.  You can see several soup-to-nuts examples of how to do this for specific environments and source data systems in the [Reference Architecture guides](../deployment_patterns/index.md).
+For production deployments you will want to define these components according to your source data systems and production environment.  This may include storing information about those components in something other than your local environment. To view implementation examples for specific environments and source data systems, see [Reference Architecture guides](https://docs.greatexpectations.io/docs/category/integrations).
```

Comment:
```suggestion
For production deployments you will want to define these components according to your source data systems and production environment.  This may include storing information about those components in something other than your local environment. To view implementation examples for specific environments and source data systems, see [Integrations](https://docs.greatexpectations.io/docs/category/integrations).
```

Since this section is no longer named "Reference Architectures" we should rename it here as well.